googletest/test/gtest_unittest.cc

```diff
@@ -2096,7 +2096,7 @@ class UnitTestRecordPropertyTestEnvironment : public Environment {
 };
 
 // This will test property recording outside of any test or test case.
-static Environment* record_property_env =
+Environment* record_property_env =
```

Comment:
Why are we removing "static"?

Reply:
Oh, that was a try to resolve the warning in the first place, but didn't change anything because this `static` is redundant anyway: Line 2099 is part of a local namespace (which has the same semantic in C++ as the `static` keyword inherited from C in this use case). The local namespace starts at line 316 and ends at line 4189.

As `static` was redundant, I left it removed (minimal refactoring ...).